client/on-demand-entries-client.js

```diff
@@ -0,0 +1,22 @@
+/* global fetch */
+
+import Router from '../lib/router'
+import 'whatwg-fetch'
+
+// Ping on every page change
+const originalOnRouteChangeComplete = Router.onRouteChangeComplete
+Router.onRouteChangeComplete = function (...args) {
+  ping()
+  originalOnRouteChangeComplete(...args)
+}
+
+// Ping every 5 seconds
+setInterval(ping, 5000)
+
+function ping () {
+  const url = `/on-demand-entries-ping?page=${Router.pathname}`
+  fetch(url)
```

Comment:
(node:18740) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
 
i am recieving this warning can you tell me how top resolve this